<HTML>
<HEAD>
  <TITLE>
  EMBASSY: VIENNA: ovrnadistance
  </TITLE>
</HEAD>
<BODY BGCOLOR="#FFFFFF" text="#000000">

<table align=center border=0 cellspacing=0 cellpadding=0>
<tr><td valign=top>
<A HREF="/" ONMOUSEOVER="self.status='Go to the EMBOSS home page';return true"><img border=0 src="emboss_icon.jpg" alt="" width=150 height=48></a>
</td>
<td align=left valign=middle>
<b><font size="+6">
ovrnadistance
</font></b>
</td></tr>
</table>
<br>&nbsp;
<p>



<H2>
Wiki
</H2>

The master copies of EMBOSS documentation are available
at <a href="http://emboss.open-bio.org/wiki/Appdocs">
http://emboss.open-bio.org/wiki/Appdocs</a>
on the EMBOSS Wiki.

<p>
Please help by correcting and extending the Wiki pages.

<H2>
    Function
</H2>
Calculate distances between RNA secondary structures
<!--
DON'T WRITE ANYTHING HERE.
IT IS DONE FOR YOU.
-->




<H2>
    Description
</H2>

<p>This is a port of the Vienna RNA package program RNAdistance.

<p>It reads RNA secondary structures and calculates one or more measures for their dissimilarity, based on tree or string editing (alignment). In addition it calculates a "base pair distance" given by the number of base pairs present in one structure, but not the other. For structures of different length base pair distance is not recommended.






<H2>
    Algorithm
</H2>

<p>RNAdistance accepts structures in bracket format, where matching brackets symbolize base pairs and unpaired bases are represented by a dot ".", or coarse grained representations where hairpins, interior loops, bulges, multiloops, stacks and external bases are represented by (H), (I), (B), (M), (S), and (E), respectively. These can be optionally weighted. Full structures can be represented in the same fashion using the identifiers (U) and (P) for unpaired and paired bases, respectively. 

<p>See the original documentation for the Vienna RNA package
http://www.tbi.univie.ac.at/~ivo/RNA/

<p>The program will continue to read new structures until a line consisting of the single character @ or an end of file condition is encountered. Input lines neither containing a valid structure nor starting with > are ignored.






<H2>
    Usage
</H2>

<!--  
	Example usage, as run from the command-line.
        Many examples illustrating different behaviours is good.
 -->

Here is a sample session with <b>ovrnadistance</b>
<p>

<p>
<table width="90%"><tr><td bgcolor="#CCFFFF"><pre>

% <b>ovrnadistance -distance 'f,h' -doalignment </b>
Calculate distances between RNA secondary structures
Vienna RNA structures file: <b>rna4.fold</b>
Vienna RNAfold output file [rna4.ovrnadistance]: <b></b>

</pre></td></tr></table><p>
<p>
<a href="#input.1">Go to the input files for this example</a><br><a href="#output.1">Go to the output files for this example</a><p><p>



<H2>
    Command line arguments
</H2>

<table CELLSPACING=0 CELLPADDING=3 BGCOLOR="#f5f5ff" ><tr><td>
<pre>
Calculate distances between RNA secondary structures
Version: EMBOSS:6.5.0.0

   Standard (Mandatory) qualifiers:
  [-structuresfile]    infile     Vienna RNA structures file
  [-outfile]           outfile    [*.ovrnadistance] Vienna RNAfold output file

   Additional (Optional) qualifiers: (none)
   Advanced (Unprompted) qualifiers:
   -distance           menu       [f] Distance types to calculate (Values: f
                                  (Full (tree)); h (HIT (tree)); w (Weighted
                                  coarse (tree)); c (Coarse (tree)); F (Full
                                  (string)); H (HIT (string)); W (Weighted
                                  coarse (string)); C (Coarse (string)); P
                                  (Base pair distance))
   -compare            menu       [p] Distance types to calculate (Values: p
                                  (Pairwise); m (Distance matrix between all
                                  structures); f (Compare all structures to
                                  the first one); c (Compare continuously))
   -schapiro           boolean    [N] Use cost matrix for comparing coarse
                                  structures
   -doalignment        boolean    [N] Alignment showing matching substructures

   Associated qualifiers:

   "-outfile" associated qualifiers
   -odirectory2        string     Output directory

   General qualifiers:
   -auto               boolean    Turn off prompts
   -stdout             boolean    Write first file to standard output
   -filter             boolean    Read first file from standard input, write
                                  first file to standard output
   -options            boolean    Prompt for standard and additional values
   -debug              boolean    Write debug output to program.dbg
   -verbose            boolean    Report some/full command line options
   -help               boolean    Report command line options and exit. More
                                  information on associated and general
                                  qualifiers can be found with -help -verbose
   -warning            boolean    Report warnings
   -error              boolean    Report errors
   -fatal              boolean    Report fatal errors
   -die                boolean    Report dying program messages
   -version            boolean    Report version number and exit

</pre>
</td></tr></table>
<P>
<table border cellspacing=0 cellpadding=3 bgcolor="#ccccff">
<tr bgcolor="#FFFFCC">
<th align="left">Qualifier</th>
<th align="left">Type</th>
<th align="left">Description</th>
<th align="left">Allowed values</th>
<th align="left">Default</th>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Standard (Mandatory) qualifiers</th>
</tr>

<tr bgcolor="#FFFFCC">
<td>[-structuresfile]<br>(Parameter 1)</td>
<td>infile</td>
<td>Vienna RNA structures file</td>
<td>Input file</td>
<td><b>Required</b></td>
</tr>

<tr bgcolor="#FFFFCC">
<td>[-outfile]<br>(Parameter 2)</td>
<td>outfile</td>
<td>Vienna RNAfold output file</td>
<td>Output file</td>
<td><i>&lt;*&gt;</i>.ovrnadistance</td>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Additional (Optional) qualifiers</th>
</tr>

<tr>
<td colspan=5>(none)</td>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Advanced (Unprompted) qualifiers</th>
</tr>

<tr bgcolor="#FFFFCC">
<td>-distance</td>
<td>list</td>
<td>Distance types to calculate</td>
<td><table><tr><td>f</td> <td><i>(Full (tree))</i></td></tr><tr><td>h</td> <td><i>(HIT (tree))</i></td></tr><tr><td>w</td> <td><i>(Weighted coarse (tree))</i></td></tr><tr><td>c</td> <td><i>(Coarse (tree))</i></td></tr><tr><td>F</td> <td><i>(Full (string))</i></td></tr><tr><td>H</td> <td><i>(HIT (string))</i></td></tr><tr><td>W</td> <td><i>(Weighted coarse (string))</i></td></tr><tr><td>C</td> <td><i>(Coarse (string))</i></td></tr><tr><td>P</td> <td><i>(Base pair distance)</i></td></tr></table></td>
<td>f</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-compare</td>
<td>list</td>
<td>Distance types to calculate</td>
<td><table><tr><td>p</td> <td><i>(Pairwise)</i></td></tr><tr><td>m</td> <td><i>(Distance matrix between all structures)</i></td></tr><tr><td>f</td> <td><i>(Compare all structures to the first one)</i></td></tr><tr><td>c</td> <td><i>(Compare continuously)</i></td></tr></table></td>
<td>p</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-schapiro</td>
<td>boolean</td>
<td>Use cost matrix for comparing coarse structures</td>
<td>Boolean value Yes/No</td>
<td>No</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-doalignment</td>
<td>boolean</td>
<td>Alignment showing matching substructures</td>
<td>Boolean value Yes/No</td>
<td>No</td>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Associated qualifiers</th>
</tr>

<tr bgcolor="#FFFFCC">
<td align="left" colspan=5>"-outfile" associated outfile qualifiers
</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -odirectory2<br>-odirectory_outfile</td>
<td>string</td>
<td>Output directory</td>
<td>Any string</td>
<td>&nbsp;</td>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>General qualifiers</th>
</tr>

<tr bgcolor="#FFFFCC">
<td> -auto</td>
<td>boolean</td>
<td>Turn off prompts</td>
<td>Boolean value Yes/No</td>
<td>N</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -stdout</td>
<td>boolean</td>
<td>Write first file to standard output</td>
<td>Boolean value Yes/No</td>
<td>N</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -filter</td>
<td>boolean</td>
<td>Read first file from standard input, write first file to standard output</td>
<td>Boolean value Yes/No</td>
<td>N</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -options</td>
<td>boolean</td>
<td>Prompt for standard and additional values</td>
<td>Boolean value Yes/No</td>
<td>N</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -debug</td>
<td>boolean</td>
<td>Write debug output to program.dbg</td>
<td>Boolean value Yes/No</td>
<td>N</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -verbose</td>
<td>boolean</td>
<td>Report some/full command line options</td>
<td>Boolean value Yes/No</td>
<td>Y</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -help</td>
<td>boolean</td>
<td>Report command line options and exit. More information on associated and general qualifiers can be found with -help -verbose</td>
<td>Boolean value Yes/No</td>
<td>N</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -warning</td>
<td>boolean</td>
<td>Report warnings</td>
<td>Boolean value Yes/No</td>
<td>Y</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -error</td>
<td>boolean</td>
<td>Report errors</td>
<td>Boolean value Yes/No</td>
<td>Y</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -fatal</td>
<td>boolean</td>
<td>Report fatal errors</td>
<td>Boolean value Yes/No</td>
<td>Y</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -die</td>
<td>boolean</td>
<td>Report dying program messages</td>
<td>Boolean value Yes/No</td>
<td>Y</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -version</td>
<td>boolean</td>
<td>Report version number and exit</td>
<td>Boolean value Yes/No</td>
<td>N</td>
</tr>

</table>

<!--
DON'T WRITE ANYTHING HERE.
IT IS DONE FOR YOU.
-->








<H2>
    Input file format
</H2>

<!-- 
        This includes example input file formats.
        This should be a detailed description and example - assume
        someone will want to parse this file and will want to know what
        happens in unusual cases - null input, etc. 
   -->

<b>ovrnadistance</b> reads any normal sequence USAs.

<p>


<a name="input.1"></a>
<h3>Input files for usage example </h3>
<p><h3>File: rna4.fold</h3>
<table width="90%"><tr><td bgcolor="#FFCCFF">
<pre>
((.(((((((.....))))))).))....((..((((.....)))).)).
.....((((..((((..........)))).)))).....(((....))).
</pre>
</td></tr></table><p>





<H2>
    Output file format
</H2>

<!-- 
        This includes example output file formats.
        This should be a detailed description and example - assume
        someone will want to parse this file and will want to know what
        happens in unusual cases - null output, errors etc. 

        If you wish to include the standard description of the avalable
	report formats, use:
#include file="inc/reportformats.ihtml"
   -->

<b>ovrnadistance</b> 
outputs a graph to the specified graphics device. 
outputs a report format file. The default format is ...


<p>


<a name="output.1"></a>
<h3>Output files for usage example </h3>
<p><h3>File: rna4.ovrnadistance</h3>
<table width="90%"><tr><td bgcolor="#CCFFCC">
<pre>
f: 26  
(_____(.(((__((((....._____))))_))).))....((..((((.....)))).)).
_.....(_(((..((((..........)))).)))_)_....__.__(((...._)))_.___
h: 32  
(____((U1)((U5_)P7)(U1)P2)(U4)((U2)((U5)P4)(U1)P2)(U1)R1)
((U5)((U2)((U10)P4)(U1)P4)(U5)_____((U4)P3)(U1)_______R1)

</pre>
</td></tr></table><p>





<H2>
    Data files
</H2>

For details of Vienna RNA file formats, see the original documentation
http://www.tbi.univie.ac.at/~ivo/RNA/


<H2>
    Notes
</H2>

<!-- 
        Restrictions.
        Interesting behaviour.
        Useful things you can do with this program.
   -->

None.







<H2>
    References
</H2>

<!-- 
        Bibliography for methods used.
   -->

None.








<H2>
    Warnings
</H2>

<!-- 
        Potentially stupid things the program will let you do.
   -->

None.







<H2>
    Diagnostic Error Messages
</H2>

<!-- 
        Error messages specific to this program, eg:
        "FATAL xxx" - means you have not set up the xxx data using program <b>prog</b>.<p>
   -->

None.







<H2>
    Exit status
</H2>

<!-- 
        Description of the exit status for various error conditions
   -->

It always exits with status 0.








<H2>
    Known bugs
</H2>


<!-- 
        Bugs noted but not yet fixed.
   -->

None.








<!--
<H2>
    See also
</H2>
-->
<h2><a name="See also">See also</a></h2>
<table border cellpadding=4 bgcolor="#FFFFF0">
<tr><th>Program name</th>
<th>Description</th></tr>
<tr>
<td><a href="/emboss/apps/banana.html">banana</a></td>
<td>Plot bending and curvature data for B-DNA</td>
</tr>

<tr>
<td><a href="/emboss/apps/btwisted.html">btwisted</a></td>
<td>Calculate the twisting in a B-DNA sequence</td>
</tr>

<tr>
<td><a href="/emboss/apps/einverted.html">einverted</a></td>
<td>Find inverted repeats in nucleotide sequences</td>
</tr>

<tr>
<td><a href="ovrnaalifold.html">ovrnaalifold</a></td>
<td>Calculate secondary structures for a set of aligned RNAs</td>
</tr>

<tr>
<td><a href="ovrnaalifoldpf.html">ovrnaalifoldpf</a></td>
<td>Calculate secondary structures for a set of aligned RNAs (partition)</td>
</tr>

<tr>
<td><a href="ovrnacofold.html">ovrnacofold</a></td>
<td>Calculate secondary structures of RNA dimers</td>
</tr>

<tr>
<td><a href="ovrnacofoldconc.html">ovrnacofoldconc</a></td>
<td>Calculate secondary structures of RNA dimers (concentrations)</td>
</tr>

<tr>
<td><a href="ovrnacofoldpf.html">ovrnacofoldpf</a></td>
<td>Calculate secondary structures of RNA dimers (partitioning)</td>
</tr>

<tr>
<td><a href="ovrnaduplex.html">ovrnaduplex</a></td>
<td>Predict RNA duplex (hybridization) sites and structure</td>
</tr>

<tr>
<td><a href="ovrnaeval.html">ovrnaeval</a></td>
<td>Calculate energy of RNA sequences with a given secondary structure</td>
</tr>

<tr>
<td><a href="ovrnaevalpair.html">ovrnaevalpair</a></td>
<td>Calculate energy of RNA sequences on given secondary structure</td>
</tr>

<tr>
<td><a href="ovrnafold.html">ovrnafold</a></td>
<td>Calculate min. energy RNA structure / pair probabilities (partition)</td>
</tr>

<tr>
<td><a href="ovrnafoldpf.html">ovrnafoldpf</a></td>
<td>Calculate min. energy RNA structure / pair probabilities</td>
</tr>

<tr>
<td><a href="ovrnaheat.html">ovrnaheat</a></td>
<td>Calculate specific heat of RNA melting</td>
</tr>

<tr>
<td><a href="ovrnainverse.html">ovrnainverse</a></td>
<td>Find RNA sequences with a given secondary structure</td>
</tr>

<tr>
<td><a href="ovrnalfold.html">ovrnalfold</a></td>
<td>Calculate locally stable secondary structures of RNAs</td>
</tr>

<tr>
<td><a href="ovrnaplot.html">ovrnaplot</a></td>
<td>Draw RNA secondary structures</td>
</tr>

<tr>
<td><a href="ovrnasubopt.html">ovrnasubopt</a></td>
<td>Calculate suboptimal secondary structure of RNA</td>
</tr>

<tr>
<td><a href="/emboss/apps/sirna.html">sirna</a></td>
<td>Find siRNA duplexes in mRNA</td>
</tr>

<tr>
<td><a href="/embassy/vienna2/vrna2dfold.html">vrna2dfold</a></td>
<td>Calculate RNA structures and samples of k,l neighbourhoods</td>
</tr>

<tr>
<td><a href="/embassy/vienna2/vrnaaliduplex.html">vrnaaliduplex</a></td>
<td>RNA duplex calculation for two sequence alignments</td>
</tr>

<tr>
<td><a href="/embassy/vienna2/vrnaalifold.html">vrnaalifold</a></td>
<td>Calculate secondary structures for a set of aligned RNAs</td>
</tr>

<tr>
<td><a href="/embassy/vienna2/vrnaalifoldpf.html">vrnaalifoldpf</a></td>
<td>Calculate secondary structures for a set of aligned RNAs (partition)</td>
</tr>

<tr>
<td><a href="/embassy/vienna2/vrnacofold.html">vrnacofold</a></td>
<td>Calculate secondary structures of RNA dimers</td>
</tr>

<tr>
<td><a href="/embassy/vienna2/vrnacofoldconc.html">vrnacofoldconc</a></td>
<td>Calculate secondary structures of RNA dimers (concentrations)</td>
</tr>

<tr>
<td><a href="/embassy/vienna2/vrnacofoldpf.html">vrnacofoldpf</a></td>
<td>Calculate secondary structures of RNA dimers (partitioning)</td>
</tr>

<tr>
<td><a href="/embassy/vienna2/vrnadistance.html">vrnadistance</a></td>
<td>Calculate distances between RNA secondary structures</td>
</tr>

<tr>
<td><a href="/embassy/vienna2/vrnaduplex.html">vrnaduplex</a></td>
<td>Predict RNA duplex (hybridization) sites and structure</td>
</tr>

<tr>
<td><a href="/embassy/vienna2/vrnaeval.html">vrnaeval</a></td>
<td>Calculate energy of RNA sequences with a given secondary structure</td>
</tr>

<tr>
<td><a href="/embassy/vienna2/vrnaevalpair.html">vrnaevalpair</a></td>
<td>Calculate energy of RNA sequences on given secondary structure</td>
</tr>

<tr>
<td><a href="/embassy/vienna2/vrnafold.html">vrnafold</a></td>
<td>Calculate min. energy RNA secondary structures and pair probabilities</td>
</tr>

<tr>
<td><a href="/embassy/vienna2/vrnafoldpf.html">vrnafoldpf</a></td>
<td>Calculate min. energy RNA structures / pair probabilities (partition)</td>
</tr>

<tr>
<td><a href="/embassy/vienna2/vrnaheat.html">vrnaheat</a></td>
<td>Calculate specific heat of RNA melting</td>
</tr>

<tr>
<td><a href="/embassy/vienna2/vrnainverse.html">vrnainverse</a></td>
<td>Find RNA sequences with a given secondary structure</td>
</tr>

<tr>
<td><a href="/embassy/vienna2/vrnalalifoldpf.html">vrnalalifoldpf</a></td>
<td>Calculate secondary structures for a set of aligned RNAs (partition)</td>
</tr>

<tr>
<td><a href="/embassy/vienna2/vrnalfold.html">vrnalfold</a></td>
<td>Calculate locally stable secondary structures of RNAs</td>
</tr>

<tr>
<td><a href="/embassy/vienna2/vrnalfoldz.html">vrnalfoldz</a></td>
<td>Calculate locally stable secondary structures of RNAs plus zscore</td>
</tr>

<tr>
<td><a href="/embassy/vienna2/vrnapkplex.html">vrnapkplex</a></td>
<td>Calculate RNA structures plus pseudoknots</td>
</tr>

<tr>
<td><a href="/embassy/vienna2/vrnaplfold.html">vrnaplfold</a></td>
<td>Compute avg. pair probabilities for local base pairs in RNA sequences</td>
</tr>

<tr>
<td><a href="/embassy/vienna2/vrnaplot.html">vrnaplot</a></td>
<td>Draw RNA secondary structures</td>
</tr>

<tr>
<td><a href="/embassy/vienna2/vrnasubopt.html">vrnasubopt</a></td>
<td>Calculate suboptimal secondary structures of RNAs</td>
</tr>

</table>
<!-- 
        Add any comments about other associated programs (to prepare
        data files?) that seealso doesn't find. 
   -->










<H2>
    Author(s)
</H2>

This program is an EMBOSS conversion of a program written by Ivo Hofacker
as part of his VIENNA package.

<p>
Please report all bugs to the EMBOSS bug team (emboss-bug&nbsp;&copy;&nbsp;emboss.open-bio.org) not to the original author.

<H2>
    History
</H2>

Converted (October 2005) by Alan Bleasby



<H2>
    Target users
</H2>

<!--
        For general users, requested by one user, for EMBOSS site
        maintainers, for EMBOSS developers etc.
        eg:
        "This program is intended to be used by everyone and everything,
        from naive users to embedded scripts." 
	Which is easy to include using:
   -->

This program is intended to be used by everyone and everything, from naive users to embedded scripts.


<H2>
    Comments
</H2>


</BODY>
</HTML>

