<HTML>
<HEAD>
  <TITLE>
  EMBASSY: VIENNA: ovrnaevalpair
  </TITLE>
</HEAD>
<BODY BGCOLOR="#FFFFFF" text="#000000">

<table align=center border=0 cellspacing=0 cellpadding=0>
<tr><td valign=top>
<A HREF="/" ONMOUSEOVER="self.status='Go to the EMBOSS home page';return true"><img border=0 src="emboss_icon.jpg" alt="" width=150 height=48></a>
</td>
<td align=left valign=middle>
<b><font size="+6">
ovrnaevalpair
</font></b>
</td></tr>
</table>
<br>&nbsp;
<p>



<H2>
Wiki
</H2>

The master copies of EMBOSS documentation are available
at <a href="http://emboss.open-bio.org/wiki/Appdocs">
http://emboss.open-bio.org/wiki/Appdocs</a>
on the EMBOSS Wiki.

<p>
Please help by correcting and extending the Wiki pages.

<H2>
    Function
</H2>
Calculate energy of RNA sequences on given secondary structure
<!--
DON'T WRITE ANYTHING HERE.
IT IS DONE FOR YOU.
-->




<H2>
    Description
</H2>

<p>This is a port of the Vienna RNA package program RNAeval.

<p>It evaluates the free energy of the co-folding of two RNA strands with fixed secondary structure. Sequences and structures are read and the energy in Kcal/Mol is output.


<p>
The original program has additional inputs and produces different
outputs, depending on the options selected. In the EMBASSY
implementation it is split into
<b>ovrnaeval</b> and <b>ovrnaevalpair</b>



<H2>
    Algorithm
</H2>

The program will continue to read new sequences and structures until a line consisting of the single character "@" or an end of file condition is encountered.

<p>See the original documentation for the Vienna RNA package
http://www.tbi.univie.ac.at/~ivo/RNA/



<H2>
    Usage
</H2>

<!--  
	Example usage, as run from the command-line.
        Many examples illustrating different behaviours is good.
 -->

Here is a sample session with <b>ovrnaevalpair</b>
<p>

<p>
<table width="90%"><tr><td bgcolor="#CCFFFF"><pre>

% <b>ovrnaevalpair </b>
Calculate energy of RNA sequences on given secondary structure
Input nucleotide sequence: <b>cofold-red.fasta</b>
Vienna RNA structure constraints file: <b>cofold-red.con</b>
Second nucleotide sequence: <b>cofold-blue.fasta</b>
Second vienna RNA structure constraints file: <b>cofold-blue.con</b>
Vienna RNAeval output file [red.ovrnaevalpair]: <b></b>

</pre></td></tr></table><p>
<p>
<a href="#input.1">Go to the input files for this example</a><br><a href="#output.1">Go to the output files for this example</a><p><p>



<H2>
    Command line arguments
</H2>

<table CELLSPACING=0 CELLPADDING=3 BGCOLOR="#f5f5ff" ><tr><td>
<pre>
Calculate energy of RNA sequences on given secondary structure
Version: EMBOSS:6.5.0.0

   Standard (Mandatory) qualifiers:
  [-asequence]         sequence   Nucleotide sequence filename and optional
                                  format, or reference (input USA)
  [-aconstraintfile]   infile     Vienna RNA structure constraints file
  [-bsequence]         sequence   Nucleotide sequence filename and optional
                                  format, or reference (input USA)
  [-bconstraintfile]   infile     Second vienna RNA structure constraints file
  [-outfile]           outfile    [*.ovrnaevalpair] Vienna RNAeval output file

   Additional (Optional) qualifiers: (none)
   Advanced (Unprompted) qualifiers:
   -paramfile          infile     Vienna RNA parameters file (optional)
   -temperature        float      [37.0] Temperature (Any numeric value)
   -[no]convert        boolean    [Y] Convert T to U
   -[no]tetraloop      boolean    [Y] Stabilising energies for tetra-loops
   -circular           boolean    [N] Circular RNA
   -energy             menu       [0] Rarely used option to fold sequences
                                  from the ABCD... alphabet (Values: 0 (BP); 1
                                  (Any with GC); 2 (Any with AU parameters))
   -dangles            menu       [1] Method (Values: 0 (No stabilising
                                  energies for dangling ends); 1 (Use
                                  stabilising energies for dangling ends); 2
                                  (Dangling energies as in a partition
                                  function 3:Allow coaxial stacking of
                                  adjacent helices))
   -logml              boolean    [N] Use log scale for multiloop energies

   Associated qualifiers:

   "-asequence" associated qualifiers
   -sbegin1            integer    Start of the sequence to be used
   -send1              integer    End of the sequence to be used
   -sreverse1          boolean    Reverse (if DNA)
   -sask1              boolean    Ask for begin/end/reverse
   -snucleotide1       boolean    Sequence is nucleotide
   -sprotein1          boolean    Sequence is protein
   -slower1            boolean    Make lower case
   -supper1            boolean    Make upper case
   -scircular1         boolean    Sequence is circular
   -sformat1           string     Input sequence format
   -iquery1            string     Input query fields or ID list
   -ioffset1           integer    Input start position offset
   -sdbname1           string     Database name
   -sid1               string     Entryname
   -ufo1               string     UFO features
   -fformat1           string     Features format
   -fopenfile1         string     Features file name

   "-bsequence" associated qualifiers
   -sbegin3            integer    Start of the sequence to be used
   -send3              integer    End of the sequence to be used
   -sreverse3          boolean    Reverse (if DNA)
   -sask3              boolean    Ask for begin/end/reverse
   -snucleotide3       boolean    Sequence is nucleotide
   -sprotein3          boolean    Sequence is protein
   -slower3            boolean    Make lower case
   -supper3            boolean    Make upper case
   -scircular3         boolean    Sequence is circular
   -sformat3           string     Input sequence format
   -iquery3            string     Input query fields or ID list
   -ioffset3           integer    Input start position offset
   -sdbname3           string     Database name
   -sid3               string     Entryname
   -ufo3               string     UFO features
   -fformat3           string     Features format
   -fopenfile3         string     Features file name

   "-outfile" associated qualifiers
   -odirectory5        string     Output directory

   General qualifiers:
   -auto               boolean    Turn off prompts
   -stdout             boolean    Write first file to standard output
   -filter             boolean    Read first file from standard input, write
                                  first file to standard output
   -options            boolean    Prompt for standard and additional values
   -debug              boolean    Write debug output to program.dbg
   -verbose            boolean    Report some/full command line options
   -help               boolean    Report command line options and exit. More
                                  information on associated and general
                                  qualifiers can be found with -help -verbose
   -warning            boolean    Report warnings
   -error              boolean    Report errors
   -fatal              boolean    Report fatal errors
   -die                boolean    Report dying program messages
   -version            boolean    Report version number and exit

</pre>
</td></tr></table>
<P>
<table border cellspacing=0 cellpadding=3 bgcolor="#ccccff">
<tr bgcolor="#FFFFCC">
<th align="left">Qualifier</th>
<th align="left">Type</th>
<th align="left">Description</th>
<th align="left">Allowed values</th>
<th align="left">Default</th>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Standard (Mandatory) qualifiers</th>
</tr>

<tr bgcolor="#FFFFCC">
<td>[-asequence]<br>(Parameter 1)</td>
<td>sequence</td>
<td>Nucleotide sequence filename and optional format, or reference (input USA)</td>
<td>Readable sequence</td>
<td><b>Required</b></td>
</tr>

<tr bgcolor="#FFFFCC">
<td>[-aconstraintfile]<br>(Parameter 2)</td>
<td>infile</td>
<td>Vienna RNA structure constraints file</td>
<td>Input file</td>
<td><b>Required</b></td>
</tr>

<tr bgcolor="#FFFFCC">
<td>[-bsequence]<br>(Parameter 3)</td>
<td>sequence</td>
<td>Nucleotide sequence filename and optional format, or reference (input USA)</td>
<td>Readable sequence</td>
<td><b>Required</b></td>
</tr>

<tr bgcolor="#FFFFCC">
<td>[-bconstraintfile]<br>(Parameter 4)</td>
<td>infile</td>
<td>Second vienna RNA structure constraints file</td>
<td>Input file</td>
<td><b>Required</b></td>
</tr>

<tr bgcolor="#FFFFCC">
<td>[-outfile]<br>(Parameter 5)</td>
<td>outfile</td>
<td>Vienna RNAeval output file</td>
<td>Output file</td>
<td><i>&lt;*&gt;</i>.ovrnaevalpair</td>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Additional (Optional) qualifiers</th>
</tr>

<tr>
<td colspan=5>(none)</td>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Advanced (Unprompted) qualifiers</th>
</tr>

<tr bgcolor="#FFFFCC">
<td>-paramfile</td>
<td>infile</td>
<td>Vienna RNA parameters file (optional)</td>
<td>Input file</td>
<td><b>Required</b></td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-temperature</td>
<td>float</td>
<td>Temperature</td>
<td>Any numeric value</td>
<td>37.0</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-[no]convert</td>
<td>boolean</td>
<td>Convert T to U</td>
<td>Boolean value Yes/No</td>
<td>Yes</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-[no]tetraloop</td>
<td>boolean</td>
<td>Stabilising energies for tetra-loops</td>
<td>Boolean value Yes/No</td>
<td>Yes</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-circular</td>
<td>boolean</td>
<td>Circular RNA</td>
<td>Boolean value Yes/No</td>
<td>No</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-energy</td>
<td>list</td>
<td>Rarely used option to fold sequences from the ABCD... alphabet</td>
<td><table><tr><td>0</td> <td><i>(BP)</i></td></tr><tr><td>1</td> <td><i>(Any with GC)</i></td></tr><tr><td>2</td> <td><i>(Any with AU parameters)</i></td></tr></table></td>
<td>0</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-dangles</td>
<td>list</td>
<td>Method</td>
<td><table><tr><td>0</td> <td><i>(No stabilising energies for dangling ends)</i></td></tr><tr><td>1</td> <td><i>(Use stabilising energies for dangling ends)</i></td></tr><tr><td>2</td> <td><i>(Dangling energies as in a partition function 3:Allow coaxial stacking of adjacent helices)</i></td></tr></table></td>
<td>1</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-logml</td>
<td>boolean</td>
<td>Use log scale for multiloop energies</td>
<td>Boolean value Yes/No</td>
<td>No</td>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Associated qualifiers</th>
</tr>

<tr bgcolor="#FFFFCC">
<td align="left" colspan=5>"-asequence" associated sequence qualifiers
</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -sbegin1<br>-sbegin_asequence</td>
<td>integer</td>
<td>Start of the sequence to be used</td>
<td>Any integer value</td>
<td>0</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -send1<br>-send_asequence</td>
<td>integer</td>
<td>End of the sequence to be used</td>
<td>Any integer value</td>
<td>0</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -sreverse1<br>-sreverse_asequence</td>
<td>boolean</td>
<td>Reverse (if DNA)</td>
<td>Boolean value Yes/No</td>
<td>N</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -sask1<br>-sask_asequence</td>
<td>boolean</td>
<td>Ask for begin/end/reverse</td>
<td>Boolean value Yes/No</td>
<td>N</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -snucleotide1<br>-snucleotide_asequence</td>
<td>boolean</td>
<td>Sequence is nucleotide</td>
<td>Boolean value Yes/No</td>
<td>N</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -sprotein1<br>-sprotein_asequence</td>
<td>boolean</td>
<td>Sequence is protein</td>
<td>Boolean value Yes/No</td>
<td>N</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -slower1<br>-slower_asequence</td>
<td>boolean</td>
<td>Make lower case</td>
<td>Boolean value Yes/No</td>
<td>N</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -supper1<br>-supper_asequence</td>
<td>boolean</td>
<td>Make upper case</td>
<td>Boolean value Yes/No</td>
<td>N</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -scircular1<br>-scircular_asequence</td>
<td>boolean</td>
<td>Sequence is circular</td>
<td>Boolean value Yes/No</td>
<td>N</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -sformat1<br>-sformat_asequence</td>
<td>string</td>
<td>Input sequence format</td>
<td>Any string</td>
<td>&nbsp;</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -iquery1<br>-iquery_asequence</td>
<td>string</td>
<td>Input query fields or ID list</td>
<td>Any string</td>
<td>&nbsp;</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -ioffset1<br>-ioffset_asequence</td>
<td>integer</td>
<td>Input start position offset</td>
<td>Any integer value</td>
<td>0</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -sdbname1<br>-sdbname_asequence</td>
<td>string</td>
<td>Database name</td>
<td>Any string</td>
<td>&nbsp;</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -sid1<br>-sid_asequence</td>
<td>string</td>
<td>Entryname</td>
<td>Any string</td>
<td>&nbsp;</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -ufo1<br>-ufo_asequence</td>
<td>string</td>
<td>UFO features</td>
<td>Any string</td>
<td>&nbsp;</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -fformat1<br>-fformat_asequence</td>
<td>string</td>
<td>Features format</td>
<td>Any string</td>
<td>&nbsp;</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -fopenfile1<br>-fopenfile_asequence</td>
<td>string</td>
<td>Features file name</td>
<td>Any string</td>
<td>&nbsp;</td>
</tr>

<tr bgcolor="#FFFFCC">
<td align="left" colspan=5>"-bsequence" associated sequence qualifiers
</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -sbegin3<br>-sbegin_bsequence</td>
<td>integer</td>
<td>Start of the sequence to be used</td>
<td>Any integer value</td>
<td>0</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -send3<br>-send_bsequence</td>
<td>integer</td>
<td>End of the sequence to be used</td>
<td>Any integer value</td>
<td>0</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -sreverse3<br>-sreverse_bsequence</td>
<td>boolean</td>
<td>Reverse (if DNA)</td>
<td>Boolean value Yes/No</td>
<td>N</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -sask3<br>-sask_bsequence</td>
<td>boolean</td>
<td>Ask for begin/end/reverse</td>
<td>Boolean value Yes/No</td>
<td>N</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -snucleotide3<br>-snucleotide_bsequence</td>
<td>boolean</td>
<td>Sequence is nucleotide</td>
<td>Boolean value Yes/No</td>
<td>N</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -sprotein3<br>-sprotein_bsequence</td>
<td>boolean</td>
<td>Sequence is protein</td>
<td>Boolean value Yes/No</td>
<td>N</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -slower3<br>-slower_bsequence</td>
<td>boolean</td>
<td>Make lower case</td>
<td>Boolean value Yes/No</td>
<td>N</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -supper3<br>-supper_bsequence</td>
<td>boolean</td>
<td>Make upper case</td>
<td>Boolean value Yes/No</td>
<td>N</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -scircular3<br>-scircular_bsequence</td>
<td>boolean</td>
<td>Sequence is circular</td>
<td>Boolean value Yes/No</td>
<td>N</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -sformat3<br>-sformat_bsequence</td>
<td>string</td>
<td>Input sequence format</td>
<td>Any string</td>
<td>&nbsp;</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -iquery3<br>-iquery_bsequence</td>
<td>string</td>
<td>Input query fields or ID list</td>
<td>Any string</td>
<td>&nbsp;</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -ioffset3<br>-ioffset_bsequence</td>
<td>integer</td>
<td>Input start position offset</td>
<td>Any integer value</td>
<td>0</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -sdbname3<br>-sdbname_bsequence</td>
<td>string</td>
<td>Database name</td>
<td>Any string</td>
<td>&nbsp;</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -sid3<br>-sid_bsequence</td>
<td>string</td>
<td>Entryname</td>
<td>Any string</td>
<td>&nbsp;</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -ufo3<br>-ufo_bsequence</td>
<td>string</td>
<td>UFO features</td>
<td>Any string</td>
<td>&nbsp;</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -fformat3<br>-fformat_bsequence</td>
<td>string</td>
<td>Features format</td>
<td>Any string</td>
<td>&nbsp;</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -fopenfile3<br>-fopenfile_bsequence</td>
<td>string</td>
<td>Features file name</td>
<td>Any string</td>
<td>&nbsp;</td>
</tr>

<tr bgcolor="#FFFFCC">
<td align="left" colspan=5>"-outfile" associated outfile qualifiers
</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -odirectory5<br>-odirectory_outfile</td>
<td>string</td>
<td>Output directory</td>
<td>Any string</td>
<td>&nbsp;</td>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>General qualifiers</th>
</tr>

<tr bgcolor="#FFFFCC">
<td> -auto</td>
<td>boolean</td>
<td>Turn off prompts</td>
<td>Boolean value Yes/No</td>
<td>N</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -stdout</td>
<td>boolean</td>
<td>Write first file to standard output</td>
<td>Boolean value Yes/No</td>
<td>N</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -filter</td>
<td>boolean</td>
<td>Read first file from standard input, write first file to standard output</td>
<td>Boolean value Yes/No</td>
<td>N</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -options</td>
<td>boolean</td>
<td>Prompt for standard and additional values</td>
<td>Boolean value Yes/No</td>
<td>N</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -debug</td>
<td>boolean</td>
<td>Write debug output to program.dbg</td>
<td>Boolean value Yes/No</td>
<td>N</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -verbose</td>
<td>boolean</td>
<td>Report some/full command line options</td>
<td>Boolean value Yes/No</td>
<td>Y</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -help</td>
<td>boolean</td>
<td>Report command line options and exit. More information on associated and general qualifiers can be found with -help -verbose</td>
<td>Boolean value Yes/No</td>
<td>N</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -warning</td>
<td>boolean</td>
<td>Report warnings</td>
<td>Boolean value Yes/No</td>
<td>Y</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -error</td>
<td>boolean</td>
<td>Report errors</td>
<td>Boolean value Yes/No</td>
<td>Y</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -fatal</td>
<td>boolean</td>
<td>Report fatal errors</td>
<td>Boolean value Yes/No</td>
<td>Y</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -die</td>
<td>boolean</td>
<td>Report dying program messages</td>
<td>Boolean value Yes/No</td>
<td>Y</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -version</td>
<td>boolean</td>
<td>Report version number and exit</td>
<td>Boolean value Yes/No</td>
<td>N</td>
</tr>

</table>

<!--
DON'T WRITE ANYTHING HERE.
IT IS DONE FOR YOU.
-->








<H2>
    Input file format
</H2>

<!-- 
        This includes example input file formats.
        This should be a detailed description and example - assume
        someone will want to parse this file and will want to know what
        happens in unusual cases - null input, etc. 
   -->

<b>ovrnaevalpair</b> reads any normal sequence USAs.

<p>


<a name="input.1"></a>
<h3>Input files for usage example </h3>
<p><h3>File: cofold-red.fasta</h3>
<table width="90%"><tr><td bgcolor="#FFCCFF">
<pre>
&gt;red
AUGAAGAUGA
</pre>
</td></tr></table><p>
<p><h3>File: cofold-red.con</h3>
<table width="90%"><tr><td bgcolor="#FFCCFF">
<pre>
....((((..
</pre>
</td></tr></table><p>
<p><h3>File: cofold-blue.fasta</h3>
<table width="90%"><tr><td bgcolor="#FFCCFF">
<pre>
&gt;blue
CUGUCUGUCUUGAGACA
</pre>
</td></tr></table><p>
<p><h3>File: cofold-blue.con</h3>
<table width="90%"><tr><td bgcolor="#FFCCFF">
<pre>
..))))(((....))).
</pre>
</td></tr></table><p>





<H2>
    Output file format
</H2>

<!-- 
        This includes example output file formats.
        This should be a detailed description and example - assume
        someone will want to parse this file and will want to know what
        happens in unusual cases - null output, errors etc. 

        If you wish to include the standard description of the avalable
	report formats, use:
#include file="inc/reportformats.ihtml"
   -->

<b>ovrnaevalpair</b> 
outputs a graph to the specified graphics device. 
outputs a report format file. The default format is ...


<p>


<a name="output.1"></a>
<h3>Output files for usage example </h3>
<p><h3>File: red.ovrnaevalpair</h3>
<table width="90%"><tr><td bgcolor="#CCFFCC">
<pre>
AUGAAGAUGA&amp;CUGUCUGUCUUGAGACA
....((((..&amp;..))))(((....))). ( -4.40)
</pre>
</td></tr></table><p>





<H2>
    Data files
</H2>


For details of Vienna RNA file formats, see the original documentation
http://www.tbi.univie.ac.at/~ivo/RNA/



<H2>
    Notes
</H2>

<!-- 
        Restrictions.
        Interesting behaviour.
        Useful things you can do with this program.
   -->

None.







<H2>
    References
</H2>

<!-- 
        Bibliography for methods used.
   -->

None.








<H2>
    Warnings
</H2>

<!-- 
        Potentially stupid things the program will let you do.
   -->

None.







<H2>
    Diagnostic Error Messages
</H2>

<!-- 
        Error messages specific to this program, eg:
        "FATAL xxx" - means you have not set up the xxx data using program <b>prog</b>.<p>
   -->

None.







<H2>
    Exit status
</H2>

<!-- 
        Description of the exit status for various error conditions
   -->

It always exits with status 0.








<H2>
    Known bugs
</H2>


<!-- 
        Bugs noted but not yet fixed.
   -->

None.








<!--
<H2>
    See also
</H2>
-->
<h2><a name="See also">See also</a></h2>
<table border cellpadding=4 bgcolor="#FFFFF0">
<tr><th>Program name</th>
<th>Description</th></tr>
<tr>
<td><a href="/emboss/apps/banana.html">banana</a></td>
<td>Plot bending and curvature data for B-DNA</td>
</tr>

<tr>
<td><a href="/emboss/apps/btwisted.html">btwisted</a></td>
<td>Calculate the twisting in a B-DNA sequence</td>
</tr>

<tr>
<td><a href="/emboss/apps/einverted.html">einverted</a></td>
<td>Find inverted repeats in nucleotide sequences</td>
</tr>

<tr>
<td><a href="ovrnaalifold.html">ovrnaalifold</a></td>
<td>Calculate secondary structures for a set of aligned RNAs</td>
</tr>

<tr>
<td><a href="ovrnaalifoldpf.html">ovrnaalifoldpf</a></td>
<td>Calculate secondary structures for a set of aligned RNAs (partition)</td>
</tr>

<tr>
<td><a href="ovrnacofold.html">ovrnacofold</a></td>
<td>Calculate secondary structures of RNA dimers</td>
</tr>

<tr>
<td><a href="ovrnacofoldconc.html">ovrnacofoldconc</a></td>
<td>Calculate secondary structures of RNA dimers (concentrations)</td>
</tr>

<tr>
<td><a href="ovrnacofoldpf.html">ovrnacofoldpf</a></td>
<td>Calculate secondary structures of RNA dimers (partitioning)</td>
</tr>

<tr>
<td><a href="ovrnadistance.html">ovrnadistance</a></td>
<td>Calculate distances between RNA secondary structures</td>
</tr>

<tr>
<td><a href="ovrnaduplex.html">ovrnaduplex</a></td>
<td>Predict RNA duplex (hybridization) sites and structure</td>
</tr>

<tr>
<td><a href="ovrnaeval.html">ovrnaeval</a></td>
<td>Calculate energy of RNA sequences with a given secondary structure</td>
</tr>

<tr>
<td><a href="ovrnafold.html">ovrnafold</a></td>
<td>Calculate min. energy RNA structure / pair probabilities (partition)</td>
</tr>

<tr>
<td><a href="ovrnafoldpf.html">ovrnafoldpf</a></td>
<td>Calculate min. energy RNA structure / pair probabilities</td>
</tr>

<tr>
<td><a href="ovrnaheat.html">ovrnaheat</a></td>
<td>Calculate specific heat of RNA melting</td>
</tr>

<tr>
<td><a href="ovrnainverse.html">ovrnainverse</a></td>
<td>Find RNA sequences with a given secondary structure</td>
</tr>

<tr>
<td><a href="ovrnalfold.html">ovrnalfold</a></td>
<td>Calculate locally stable secondary structures of RNAs</td>
</tr>

<tr>
<td><a href="ovrnaplot.html">ovrnaplot</a></td>
<td>Draw RNA secondary structures</td>
</tr>

<tr>
<td><a href="ovrnasubopt.html">ovrnasubopt</a></td>
<td>Calculate suboptimal secondary structure of RNA</td>
</tr>

<tr>
<td><a href="/emboss/apps/sirna.html">sirna</a></td>
<td>Find siRNA duplexes in mRNA</td>
</tr>

<tr>
<td><a href="/embassy/vienna2/vrna2dfold.html">vrna2dfold</a></td>
<td>Calculate RNA structures and samples of k,l neighbourhoods</td>
</tr>

<tr>
<td><a href="/embassy/vienna2/vrnaaliduplex.html">vrnaaliduplex</a></td>
<td>RNA duplex calculation for two sequence alignments</td>
</tr>

<tr>
<td><a href="/embassy/vienna2/vrnaalifold.html">vrnaalifold</a></td>
<td>Calculate secondary structures for a set of aligned RNAs</td>
</tr>

<tr>
<td><a href="/embassy/vienna2/vrnaalifoldpf.html">vrnaalifoldpf</a></td>
<td>Calculate secondary structures for a set of aligned RNAs (partition)</td>
</tr>

<tr>
<td><a href="/embassy/vienna2/vrnacofold.html">vrnacofold</a></td>
<td>Calculate secondary structures of RNA dimers</td>
</tr>

<tr>
<td><a href="/embassy/vienna2/vrnacofoldconc.html">vrnacofoldconc</a></td>
<td>Calculate secondary structures of RNA dimers (concentrations)</td>
</tr>

<tr>
<td><a href="/embassy/vienna2/vrnacofoldpf.html">vrnacofoldpf</a></td>
<td>Calculate secondary structures of RNA dimers (partitioning)</td>
</tr>

<tr>
<td><a href="/embassy/vienna2/vrnadistance.html">vrnadistance</a></td>
<td>Calculate distances between RNA secondary structures</td>
</tr>

<tr>
<td><a href="/embassy/vienna2/vrnaduplex.html">vrnaduplex</a></td>
<td>Predict RNA duplex (hybridization) sites and structure</td>
</tr>

<tr>
<td><a href="/embassy/vienna2/vrnaeval.html">vrnaeval</a></td>
<td>Calculate energy of RNA sequences with a given secondary structure</td>
</tr>

<tr>
<td><a href="/embassy/vienna2/vrnaevalpair.html">vrnaevalpair</a></td>
<td>Calculate energy of RNA sequences on given secondary structure</td>
</tr>

<tr>
<td><a href="/embassy/vienna2/vrnafold.html">vrnafold</a></td>
<td>Calculate min. energy RNA secondary structures and pair probabilities</td>
</tr>

<tr>
<td><a href="/embassy/vienna2/vrnafoldpf.html">vrnafoldpf</a></td>
<td>Calculate min. energy RNA structures / pair probabilities (partition)</td>
</tr>

<tr>
<td><a href="/embassy/vienna2/vrnaheat.html">vrnaheat</a></td>
<td>Calculate specific heat of RNA melting</td>
</tr>

<tr>
<td><a href="/embassy/vienna2/vrnainverse.html">vrnainverse</a></td>
<td>Find RNA sequences with a given secondary structure</td>
</tr>

<tr>
<td><a href="/embassy/vienna2/vrnalalifoldpf.html">vrnalalifoldpf</a></td>
<td>Calculate secondary structures for a set of aligned RNAs (partition)</td>
</tr>

<tr>
<td><a href="/embassy/vienna2/vrnalfold.html">vrnalfold</a></td>
<td>Calculate locally stable secondary structures of RNAs</td>
</tr>

<tr>
<td><a href="/embassy/vienna2/vrnalfoldz.html">vrnalfoldz</a></td>
<td>Calculate locally stable secondary structures of RNAs plus zscore</td>
</tr>

<tr>
<td><a href="/embassy/vienna2/vrnapkplex.html">vrnapkplex</a></td>
<td>Calculate RNA structures plus pseudoknots</td>
</tr>

<tr>
<td><a href="/embassy/vienna2/vrnaplfold.html">vrnaplfold</a></td>
<td>Compute avg. pair probabilities for local base pairs in RNA sequences</td>
</tr>

<tr>
<td><a href="/embassy/vienna2/vrnaplot.html">vrnaplot</a></td>
<td>Draw RNA secondary structures</td>
</tr>

<tr>
<td><a href="/embassy/vienna2/vrnasubopt.html">vrnasubopt</a></td>
<td>Calculate suboptimal secondary structures of RNAs</td>
</tr>

</table>
<!-- 
        Add any comments about other associated programs (to prepare
        data files?) that seealso doesn't find. 
   -->










<H2>
    Author(s)
</H2>

This program is an EMBOSS conversion of a program written by Ivo Hofacker
as part of his VIENNA package.

<p>
Please report all bugs to the EMBOSS bug team (emboss-bug&nbsp;&copy;&nbsp;emboss.open-bio.org) not to the original author.

<H2>
    History
</H2>

Converted (October 2005) by Alan Bleasby



<H2>
    Target users
</H2>

<!--
        For general users, requested by one user, for EMBOSS site
        maintainers, for EMBOSS developers etc.
        eg:
        "This program is intended to be used by everyone and everything,
        from naive users to embedded scripts." 
	Which is easy to include using:
   -->

This program is intended to be used by everyone and everything, from naive users to embedded scripts.


<H2>
    Comments
</H2>


</BODY>
</HTML>

