<HTML>
<HEAD>
  <TITLE>
  EMBASSY: CLUSTALOMEGA: eomegash
  </TITLE>
</HEAD>
<BODY BGCOLOR="#FFFFFF" text="#000000">

<table align=center border=0 cellspacing=0 cellpadding=0>
<tr><td valign=top>
<A HREF="/" ONMOUSEOVER="self.status='Go to the EMBOSS home page';return true"><img border=0 src="emboss_icon.jpg" alt="" width=150 height=48></a>
</td>
<td align=left valign=middle>
<b><font size="+6">
eomegash
</font></b>
</td></tr>
</table>
<br>&nbsp;
<p>



<H2>
Wiki
</H2>

The master copies of EMBOSS documentation are available
at <a href="http://emboss.open-bio.org/wiki/Appdocs">
http://emboss.open-bio.org/wiki/Appdocs</a>
on the EMBOSS Wiki.

<p>
Please help by correcting and extending the Wiki pages.

<H2>
    Function
</H2>
Sequence with HMM (ClustalO wrapper)
<!--
DON'T WRITE ANYTHING HERE.
IT IS DONE FOR YOU.
-->




<H2>
    Description
</H2>

<!-- 
	This is the main part of the document.
        This is what the user looks at to find out if this program will do what he wants. 
        Long description.
        Why it is useful.
        When it should/shouldn't be used.
  -->

eomegash is a wrapper to clustalo. It takes a set of unaligned sequences
and one HMM and produces a combined output alignment.
<p>
Clustal-Omega (clustalo) is a general purpose multiple sequence
alignment (MSA) program for proteins. It produces high quality MSAs
and is capable of handling data-sets of hundreds of thousands of
sequences in reasonable time.
<p>
In its current form Clustal-Omega can only align protein sequences but
not DNA/RNA sequences. It is envisioned that DNA/RNA will become
available in a future version.














<H2>
    Algorithm
</H2>

<!-- 
        Algorithms
	  Pseudocode/Vague description/Handwaving/web links to useful pages
  -->

Clustal-Omega uses HMMs for the alignment engine, based on the HHalign
package from Johannes Soeding [1].  Guide trees are optionally made
using mBed [2] which can cluster very large numbers of sequences in
O(N*log(N)) time.  Multiple alignment then proceeds by aligning larger
and larger alignments using HHalign, following the clustering given by
the guide tree.









<H2>
    Usage
</H2>

<!--  
	Example usage, as run from the command-line.
        Many examples illustrating different behaviours is good.
 -->

Here is a sample session with <b>eomegash</b>
<p>

<p>
<table width="90%"><tr><td bgcolor="#CCFFFF"><pre>

% <b>eomegash ../data/globins4.hmm </b>
Sequence with HMM (ClustalO wrapper)
(aligned) output sequence set [twoglob.aln]: <b></b>
WARNING: This is a beta version of Clustal Omega, for protein only.

</pre></td></tr></table><p>
<p>
<a href="#input.1">Go to the input files for this example</a><br><a href="#output.1">Go to the output files for this example</a><p><p>



<H2>
    Command line arguments
</H2>

<table CELLSPACING=0 CELLPADDING=3 BGCOLOR="#f5f5ff" ><tr><td>
<pre>
Sequence with HMM (ClustalO wrapper)
Version: EMBOSS:6.5.0.0

   Standard (Mandatory) qualifiers:
  [-sequences]         seqset     File containing sequences to align
  [-hmm]               infile     Pre-aligned multiple sequence file (aligned
                                  columns will be kept fixed)
  [-outseq]            seqoutset  [<sequence>.<format>] Sequence set filename
                                  and optional format (output USA)

   Additional (Optional) qualifiers: (none)
   Advanced (Unprompted) qualifiers:
   -indist             infile     Pairwise distance matrix input file (skips
                                  distance computation)
   -inguide            infile     Guide tree input file (skips distance
                                  computation and guide tree clustering step)
   -dealign            toggle     [N] Dealign input sequences
   -mbed               toggle     [N] Fast, Mbed-like clustering for guide
                                  tree calculation
   -intermbed          toggle     [N] Fast, Mbed-like clustering for guide
                                  tree calculation
   -maxiterations      integer    [0] Number of (combined guide tree/HMM)
                                  iterations (Integer from 0 to 2000000000)
   -maxgiterations     integer    [2000000000] Maximum guide tree iterations
                                  (Integer from 0 to 2000000000)
   -maxhiterations     integer    [2000000000] Maximum number of HMM
                                  iterations (Integer from 0 to 2000000000)
   -maxseqs            integer    [2000000000] Maximum number of sequences
                                  (Integer from 2 to 2000000000)
   -maxlenseq          integer    [2000000000] Maximum length of sequence
                                  (Integer from 1 to 2000000000)
   -self               toggle     [N] Set options automatically (might
                                  overwrite some options
   -outdist            outfile    [*.eomegash] Pairwise distance matrix output
                                  file
   -outguide           outfile    [*.eomegash] Guide tree output file

   Associated qualifiers:

   "-sequences" associated qualifiers
   -sbegin1            integer    Start of each sequence to be used
   -send1              integer    End of each sequence to be used
   -sreverse1          boolean    Reverse (if DNA)
   -sask1              boolean    Ask for begin/end/reverse
   -snucleotide1       boolean    Sequence is nucleotide
   -sprotein1          boolean    Sequence is protein
   -slower1            boolean    Make lower case
   -supper1            boolean    Make upper case
   -scircular1         boolean    Sequence is circular
   -sformat1           string     Input sequence format
   -iquery1            string     Input query fields or ID list
   -ioffset1           integer    Input start position offset
   -sdbname1           string     Database name
   -sid1               string     Entryname
   -ufo1               string     UFO features
   -fformat1           string     Features format
   -fopenfile1         string     Features file name

   "-outseq" associated qualifiers
   -osformat3          string     Output seq format
   -osextension3       string     File name extension
   -osname3            string     Base file name
   -osdirectory3       string     Output directory
   -osdbname3          string     Database name to add
   -ossingle3          boolean    Separate file for each entry
   -oufo3              string     UFO features
   -offormat3          string     Features format
   -ofname3            string     Features file name
   -ofdirectory3       string     Output directory

   "-outdist" associated qualifiers
   -odirectory         string     Output directory

   "-outguide" associated qualifiers
   -odirectory         string     Output directory

   General qualifiers:
   -auto               boolean    Turn off prompts
   -stdout             boolean    Write first file to standard output
   -filter             boolean    Read first file from standard input, write
                                  first file to standard output
   -options            boolean    Prompt for standard and additional values
   -debug              boolean    Write debug output to program.dbg
   -verbose            boolean    Report some/full command line options
   -help               boolean    Report command line options and exit. More
                                  information on associated and general
                                  qualifiers can be found with -help -verbose
   -warning            boolean    Report warnings
   -error              boolean    Report errors
   -fatal              boolean    Report fatal errors
   -die                boolean    Report dying program messages
   -version            boolean    Report version number and exit

</pre>
</td></tr></table>
<P>
<table border cellspacing=0 cellpadding=3 bgcolor="#ccccff">
<tr bgcolor="#FFFFCC">
<th align="left">Qualifier</th>
<th align="left">Type</th>
<th align="left">Description</th>
<th align="left">Allowed values</th>
<th align="left">Default</th>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Standard (Mandatory) qualifiers</th>
</tr>

<tr bgcolor="#FFFFCC">
<td>[-sequences]<br>(Parameter 1)</td>
<td>seqset</td>
<td>File containing sequences to align</td>
<td>Readable set of sequences</td>
<td><b>Required</b></td>
</tr>

<tr bgcolor="#FFFFCC">
<td>[-hmm]<br>(Parameter 2)</td>
<td>infile</td>
<td>Pre-aligned multiple sequence file (aligned columns will be kept fixed)</td>
<td>Input file</td>
<td><b>Required</b></td>
</tr>

<tr bgcolor="#FFFFCC">
<td>[-outseq]<br>(Parameter 3)</td>
<td>seqoutset</td>
<td>Sequence set filename and optional format (output USA)</td>
<td>Writeable sequences</td>
<td><i>&lt;*&gt;</i>.<i>format</i></td>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Additional (Optional) qualifiers</th>
</tr>

<tr>
<td colspan=5>(none)</td>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Advanced (Unprompted) qualifiers</th>
</tr>

<tr bgcolor="#FFFFCC">
<td>-indist</td>
<td>infile</td>
<td>Pairwise distance matrix input file (skips distance computation)</td>
<td>Input file</td>
<td><b>Required</b></td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-inguide</td>
<td>infile</td>
<td>Guide tree input file (skips distance computation and guide tree clustering step)</td>
<td>Input file</td>
<td><b>Required</b></td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-dealign</td>
<td>toggle</td>
<td>Dealign input sequences</td>
<td>Toggle value Yes/No</td>
<td>No</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-mbed</td>
<td>toggle</td>
<td>Fast, Mbed-like clustering for guide tree calculation</td>
<td>Toggle value Yes/No</td>
<td>No</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-intermbed</td>
<td>toggle</td>
<td>Fast, Mbed-like clustering for guide tree calculation</td>
<td>Toggle value Yes/No</td>
<td>No</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-maxiterations</td>
<td>integer</td>
<td>Number of (combined guide tree/HMM) iterations</td>
<td>Integer from 0 to 2000000000</td>
<td>0</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-maxgiterations</td>
<td>integer</td>
<td>Maximum guide tree iterations</td>
<td>Integer from 0 to 2000000000</td>
<td>2000000000</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-maxhiterations</td>
<td>integer</td>
<td>Maximum number of HMM iterations</td>
<td>Integer from 0 to 2000000000</td>
<td>2000000000</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-maxseqs</td>
<td>integer</td>
<td>Maximum number of sequences</td>
<td>Integer from 2 to 2000000000</td>
<td>2000000000</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-maxlenseq</td>
<td>integer</td>
<td>Maximum length of sequence</td>
<td>Integer from 1 to 2000000000</td>
<td>2000000000</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-self</td>
<td>toggle</td>
<td>Set options automatically (might overwrite some options</td>
<td>Toggle value Yes/No</td>
<td>No</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-outdist</td>
<td>outfile</td>
<td>Pairwise distance matrix output file</td>
<td>Output file</td>
<td><i>&lt;*&gt;</i>.eomegash</td>
</tr>

<tr bgcolor="#FFFFCC">
<td>-outguide</td>
<td>outfile</td>
<td>Guide tree output file</td>
<td>Output file</td>
<td><i>&lt;*&gt;</i>.eomegash</td>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>Associated qualifiers</th>
</tr>

<tr bgcolor="#FFFFCC">
<td align="left" colspan=5>"-sequences" associated seqset qualifiers
</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -sbegin1<br>-sbegin_sequences</td>
<td>integer</td>
<td>Start of each sequence to be used</td>
<td>Any integer value</td>
<td>0</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -send1<br>-send_sequences</td>
<td>integer</td>
<td>End of each sequence to be used</td>
<td>Any integer value</td>
<td>0</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -sreverse1<br>-sreverse_sequences</td>
<td>boolean</td>
<td>Reverse (if DNA)</td>
<td>Boolean value Yes/No</td>
<td>N</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -sask1<br>-sask_sequences</td>
<td>boolean</td>
<td>Ask for begin/end/reverse</td>
<td>Boolean value Yes/No</td>
<td>N</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -snucleotide1<br>-snucleotide_sequences</td>
<td>boolean</td>
<td>Sequence is nucleotide</td>
<td>Boolean value Yes/No</td>
<td>N</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -sprotein1<br>-sprotein_sequences</td>
<td>boolean</td>
<td>Sequence is protein</td>
<td>Boolean value Yes/No</td>
<td>N</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -slower1<br>-slower_sequences</td>
<td>boolean</td>
<td>Make lower case</td>
<td>Boolean value Yes/No</td>
<td>N</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -supper1<br>-supper_sequences</td>
<td>boolean</td>
<td>Make upper case</td>
<td>Boolean value Yes/No</td>
<td>N</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -scircular1<br>-scircular_sequences</td>
<td>boolean</td>
<td>Sequence is circular</td>
<td>Boolean value Yes/No</td>
<td>N</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -sformat1<br>-sformat_sequences</td>
<td>string</td>
<td>Input sequence format</td>
<td>Any string</td>
<td>&nbsp;</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -iquery1<br>-iquery_sequences</td>
<td>string</td>
<td>Input query fields or ID list</td>
<td>Any string</td>
<td>&nbsp;</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -ioffset1<br>-ioffset_sequences</td>
<td>integer</td>
<td>Input start position offset</td>
<td>Any integer value</td>
<td>0</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -sdbname1<br>-sdbname_sequences</td>
<td>string</td>
<td>Database name</td>
<td>Any string</td>
<td>&nbsp;</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -sid1<br>-sid_sequences</td>
<td>string</td>
<td>Entryname</td>
<td>Any string</td>
<td>&nbsp;</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -ufo1<br>-ufo_sequences</td>
<td>string</td>
<td>UFO features</td>
<td>Any string</td>
<td>&nbsp;</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -fformat1<br>-fformat_sequences</td>
<td>string</td>
<td>Features format</td>
<td>Any string</td>
<td>&nbsp;</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -fopenfile1<br>-fopenfile_sequences</td>
<td>string</td>
<td>Features file name</td>
<td>Any string</td>
<td>&nbsp;</td>
</tr>

<tr bgcolor="#FFFFCC">
<td align="left" colspan=5>"-outseq" associated seqoutset qualifiers
</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -osformat3<br>-osformat_outseq</td>
<td>string</td>
<td>Output seq format</td>
<td>Any string</td>
<td>&nbsp;</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -osextension3<br>-osextension_outseq</td>
<td>string</td>
<td>File name extension</td>
<td>Any string</td>
<td>&nbsp;</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -osname3<br>-osname_outseq</td>
<td>string</td>
<td>Base file name</td>
<td>Any string</td>
<td>&nbsp;</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -osdirectory3<br>-osdirectory_outseq</td>
<td>string</td>
<td>Output directory</td>
<td>Any string</td>
<td>&nbsp;</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -osdbname3<br>-osdbname_outseq</td>
<td>string</td>
<td>Database name to add</td>
<td>Any string</td>
<td>&nbsp;</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -ossingle3<br>-ossingle_outseq</td>
<td>boolean</td>
<td>Separate file for each entry</td>
<td>Boolean value Yes/No</td>
<td>N</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -oufo3<br>-oufo_outseq</td>
<td>string</td>
<td>UFO features</td>
<td>Any string</td>
<td>&nbsp;</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -offormat3<br>-offormat_outseq</td>
<td>string</td>
<td>Features format</td>
<td>Any string</td>
<td>&nbsp;</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -ofname3<br>-ofname_outseq</td>
<td>string</td>
<td>Features file name</td>
<td>Any string</td>
<td>&nbsp;</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -ofdirectory3<br>-ofdirectory_outseq</td>
<td>string</td>
<td>Output directory</td>
<td>Any string</td>
<td>&nbsp;</td>
</tr>

<tr bgcolor="#FFFFCC">
<td align="left" colspan=5>"-outdist" associated outfile qualifiers
</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -odirectory</td>
<td>string</td>
<td>Output directory</td>
<td>Any string</td>
<td>&nbsp;</td>
</tr>

<tr bgcolor="#FFFFCC">
<td align="left" colspan=5>"-outguide" associated outfile qualifiers
</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -odirectory</td>
<td>string</td>
<td>Output directory</td>
<td>Any string</td>
<td>&nbsp;</td>
</tr>

<tr bgcolor="#FFFFCC">
<th align="left" colspan=5>General qualifiers</th>
</tr>

<tr bgcolor="#FFFFCC">
<td> -auto</td>
<td>boolean</td>
<td>Turn off prompts</td>
<td>Boolean value Yes/No</td>
<td>N</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -stdout</td>
<td>boolean</td>
<td>Write first file to standard output</td>
<td>Boolean value Yes/No</td>
<td>N</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -filter</td>
<td>boolean</td>
<td>Read first file from standard input, write first file to standard output</td>
<td>Boolean value Yes/No</td>
<td>N</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -options</td>
<td>boolean</td>
<td>Prompt for standard and additional values</td>
<td>Boolean value Yes/No</td>
<td>N</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -debug</td>
<td>boolean</td>
<td>Write debug output to program.dbg</td>
<td>Boolean value Yes/No</td>
<td>N</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -verbose</td>
<td>boolean</td>
<td>Report some/full command line options</td>
<td>Boolean value Yes/No</td>
<td>Y</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -help</td>
<td>boolean</td>
<td>Report command line options and exit. More information on associated and general qualifiers can be found with -help -verbose</td>
<td>Boolean value Yes/No</td>
<td>N</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -warning</td>
<td>boolean</td>
<td>Report warnings</td>
<td>Boolean value Yes/No</td>
<td>Y</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -error</td>
<td>boolean</td>
<td>Report errors</td>
<td>Boolean value Yes/No</td>
<td>Y</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -fatal</td>
<td>boolean</td>
<td>Report fatal errors</td>
<td>Boolean value Yes/No</td>
<td>Y</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -die</td>
<td>boolean</td>
<td>Report dying program messages</td>
<td>Boolean value Yes/No</td>
<td>Y</td>
</tr>

<tr bgcolor="#FFFFCC">
<td> -version</td>
<td>boolean</td>
<td>Report version number and exit</td>
<td>Boolean value Yes/No</td>
<td>N</td>
</tr>

</table>

<!--
DON'T WRITE ANYTHING HERE.
IT IS DONE FOR YOU.
-->








<H2>
    Input file format
</H2>

<!-- 
        This includes example input file formats.
        This should be a detailed description and example - assume
        someone will want to parse this file and will want to know what
        happens in unusual cases - null input, etc. 
   -->

<b>eomegash</b> reads a set of unaligned sequences and optional distance
and guide tree files.

<p>


<a name="input.1"></a>
<h3>Input files for usage example </h3>
<p><h3>File: twoglob.fasta</h3>
<table width="90%"><tr><td bgcolor="#FFCCFF">
<pre>
&gt;HBA_HUMAN P69905 Hemoglobin subunit alpha (Alpha-globin) (Hemoglobin alpha chain)
MVLSPADKTNVKAAWGKVGAHAGEYGAEALERMFLSFPTTKTYFPHFDLSHGSAQVKGHG
KKVADALTNAVAHVDDMPNALSALSDLHAHKLRVDPVNFKLLSHCLLVTLAAHLPAEFTP
AVHASLDKFLASVSTVLTSKYR
&gt;HBA_BOVFRAG Hemoglobin subunit alpha (Alpha-globin) fragment (Hemoglobin alpha chain)
DKGNVKAAWGKVGGHAAEYGAEALERMFLSFPTTKTYFPHFDLSHGSAQVKGHG
AKVAAALTKAVEHLDDLPGALSELSDLHAHKLRVDPVNFKLLSHSLLVTLASHLPSDFTP
AVHASLDKFLANVST
</pre>
</td></tr></table><p>
<p><h3>File: globins4.hmm</h3>
<table width="90%"><tr><td bgcolor="#FFCCFF">
<pre>
HMMER3/b [3.0 | March 2010]
NAME  globins4
LENG  149
ALPH  amino
RF    no
CS    no
MAP   yes
DATE  Fri Jul 15 12:00:00 2011
NSEQ  4
EFFN  0.964844
CKSUM 2027839109
STATS LOCAL MSV       -9.9014  0.70957
STATS LOCAL VITERBI  -10.7224  0.70957
STATS LOCAL FORWARD   -4.1637  0.70957
HMM          A        C        D        E        F        G        H        I        K        L        M        N        P        Q        R        S        T        V        W        Y   
            m-&gt;m     m-&gt;i     m-&gt;d     i-&gt;m     i-&gt;i     d-&gt;m     d-&gt;d
  COMPO   2.36553  4.52577  2.96709  2.70473  3.20818  3.02239  3.41069  2.90041  2.55332  2.35210  3.67329  3.19812  3.45595  3.16091  3.07934  2.66722  2.85475  2.56965  4.55393  3.62921
          2.68640  4.42247  2.77497  2.73145  3.46376  2.40504  3.72516  3.29302  2.67763  2.69377  4.24712  2.90369  2.73719  3.18168  2.89823  2.37879  2.77497  2.98431  4.58499  3.61525
          0.57544  1.78073  1.31293  1.75577  0.18968  0.00000        *
      1   1.70038  4.17733  3.76164  3.36686  3.72281  3.29583  4.27570  2.40482  3.29230  2.54324  3.63799  3.55099  3.93183  3.61602  3.56580  2.71897  2.84104  1.67328  5.32720  4.10031      9 - -
          2.68618  4.42225  2.77519  2.73123  3.46354  2.40513  3.72494  3.29354  2.67741  2.69355  4.24690  2.90347  2.73739  3.18146  2.89801  2.37887  2.77519  2.98518  4.58477  3.61503
          0.03156  3.86736  4.58970  0.61958  0.77255  0.34406  1.23405
      2   2.62748  4.47174  3.31917  2.82619  3.63815  3.49607  2.75382  3.03401  2.75280  2.74783  3.65114  3.24714  2.62341  3.12082  3.11124  2.79244  2.89355  1.88003  5.06315  3.77128     10 - -
          2.68618  4.42225  2.77519  2.73123  3.46354  2.40513  3.72494  3.29354  2.67741  2.69355  4.24690  2.90347  2.73739  3.18146  2.89801  2.37887  2.77519  2.98518  4.58477  3.61503
          0.02321  4.17053  4.89288  0.61958  0.77255  0.48576  0.95510
      3   3.50771  4.88753  4.66754  4.31907  3.27776  4.35743  4.88268  2.50779  4.08449  0.57907  3.22569  4.56607  4.74802  4.37991  4.20749  3.97946  3.79191  2.62059  5.25407  4.04279     11 - -
          2.68618  4.42225  2.77519  2.73123  3.46354  2.40513  3.72494  3.29354  2.67741  2.69355  4.24690  2.90347  2.73739  3.18146  2.89801  2.37887  2.77519  2.98518  4.58477  3.61503
          0.02321  4.17053  4.89288  0.61958  0.77255  0.48576  0.95510
      4   2.34080  4.28719  3.51550  3.22063  4.37406  3.06195  4.29366  3.74891  3.24370  3.47337  4.31943  3.39310  3.80273  3.56072  3.55390  1.08280  2.00280  3.23325  5.72380  4.49519     12 - -
          2.68618  4.42225  2.77519  2.73123  3.46354  2.40513  3.72494  3.29354  2.67741  2.69355  4.24690  2.90347  2.73739  3.18146  2.89801  2.37887  2.77519  2.98518  4.58477  3.61503
          0.02321  4.17053  4.89288  0.61958  0.77255  0.48576  0.95510
      5   2.04218  4.87966  2.77126  1.92448  4.32563  3.33660  3.78415  3.73388  2.61886  3.34029  4.16568  2.96714  2.18012  2.94561  3.07633  2.70238  2.96803  3.36068  5.56791  4.21854     13 - -
          2.68618  4.42225  2.77519  2.73123  3.46354  2.40513  3.72494  3.29354  2.67741  2.69355  4.24690  2.90347  2.73739  3.18146  2.89801  2.37887  2.77519  2.98518  4.58477  3.61503
          0.02321  4.17053  4.89288  0.61958  0.77255  0.48576  0.95510
      6   1.67844  4.79514  2.78469  2.00544  4.34945  2.26719  3.85010  3.76260  2.72087  3.37961  4.20641  2.99947  3.85561  3.02442  3.17766  2.68798  2.96575  3.36703  5.60901  4.27005     14 - -
          2.68618  4.42225  2.77519  2.73123  3.46354  2.40513  3.72494  3.29354  2.67741  2.69355  4.24690  2.90347  2.73739  3.18146  2.89801  2.37887  2.77519  2.98518  4.58477  3.61503
          0.02321  4.17053  4.89288  0.61958  0.77255  0.48576  0.95510
      7   3.05756  5.63998  1.78612  1.06702  4.93093  3.31407  3.86695  4.45859  2.89031  3.95237  4.80915  2.77798  3.94091  3.03432  3.48933  2.93505  3.34810  4.02575  6.09427  4.58708     15 - -
          2.68618  4.42225  2.77519  2.73123  3.46354  2.40513  3.72494  3.29354  2.67741  2.69355  4.24690  2.90347  2.73739  3.18146  2.89801  2.37887  2.77519  2.98518  4.58477  3.61503
          0.02321  4.17053  4.89288  0.61958  0.77255  0.48576  0.95510
      8   2.98085  4.92154  3.50401  2.91209  3.76425  3.70161  3.75301  3.57060  1.24921  3.12012  4.04714  3.32210  4.09411  3.00014  2.54951  3.05147  3.18982  3.31354  2.52964  3.70813     16 - -
          2.68618  4.42225  2.77519  2.73123  3.46354  2.40513  3.72494  3.29354  2.67741  2.69355  4.24690  2.90347  2.73739  3.18146  2.89801  2.37887  2.77519  2.98518  4.58477  3.61503
          0.02321  4.17053  4.89288  0.61958  0.77255  0.48576  0.95510
      9   2.59353  4.69451  3.03503  2.60972  4.18917  3.33498  3.80536  3.60013  2.58953  3.21847  4.04944  3.07115  3.85428  2.20828  2.99231  2.14672  1.84753  3.23346  5.45458  4.14691     17 - -
          2.68618  4.42225  2.77519  2.73123  3.46354  2.40513  3.72494  3.29354  2.67741  2.69355  4.24690  2.90347  2.73739  3.18146  2.89801  2.37887  2.77519  2.98518  4.58477  3.61503
          0.02321  4.17053  4.89288  0.61958  0.77255  0.48576  0.95510
     10   2.23880  4.76665  3.06952  2.55248  4.02235  3.47005  3.69787  3.40277  2.01551  2.36330  3.87991  2.51243  3.88223  2.86967  2.84293  2.72413  2.90300  3.10205  5.29826  3.99496     18 - -
          2.68618  4.42225  2.77519  2.73123  3.46354  2.40513  3.72494  3.29354  2.67741  2.69355  4.24690  2.90347  2.73739  3.18146  2.89801  2.37887  2.77519  2.98518  4.58477  3.61503
          0.02321  4.17053  4.89288  0.61958  0.77255  0.48576  0.95510
     11   3.19694  4.48391  4.99800  4.51340  3.70437  4.59686  5.22508  1.39091  4.40799  2.19554  3.49619  4.71017  4.90059  4.67186  4.59646  3.99910  3.46528  0.94853  5.69937  4.48740     19 - -


<font color=red>  [Part of this file has been deleted for brevity]</font>

          2.68618  4.42225  2.77519  2.73123  3.46354  2.40513  3.72494  3.29354  2.67741  2.69355  4.24690  2.90347  2.73739  3.18146  2.89801  2.37887  2.77519  2.98518  4.58477  3.61503
          0.02321  4.17053  4.89288  0.61958  0.77255  0.48576  0.95510
    134   2.87148  5.44150  1.98118  1.74899  4.74617  3.33260  3.73791  4.25433  2.62738  3.72942  4.52187  2.07888  3.87291  2.23146  3.17389  2.78299  3.13122  3.81624  5.87307  4.40314    150 - -
          2.68618  4.42225  2.77519  2.73123  3.46354  2.40513  3.72494  3.29354  2.67741  2.69355  4.24690  2.90347  2.73739  3.18146  2.89801  2.37887  2.77519  2.98518  4.58477  3.61503
          0.02321  4.17053  4.89288  0.61958  0.77255  0.48576  0.95510
    135   3.27331  5.22456  3.46754  3.04631  4.68046  3.71770  3.96365  4.13172  0.69147  3.63653  4.59195  3.45354  4.20521  3.14539  2.53832  3.30173  3.50915  3.82642  5.61930  4.48069    151 - -
          2.68618  4.42225  2.77519  2.73123  3.46354  2.40513  3.72494  3.29354  2.67741  2.69355  4.24690  2.90347  2.73739  3.18146  2.89801  2.37887  2.77519  2.98518  4.58477  3.61503
          0.02321  4.17053  4.89288  0.61958  0.77255  0.48576  0.95510
    136   2.13728  4.25596  4.26822  3.70038  2.51560  3.93283  4.27242  2.28282  3.58922  1.68324  3.19522  3.92448  4.28790  3.80594  3.78003  3.23864  3.04401  1.82614  4.90639  3.70545    152 - -
          2.68618  4.42225  2.77519  2.73123  3.46354  2.40513  3.72494  3.29354  2.67741  2.69355  4.24690  2.90347  2.73739  3.18146  2.89801  2.37887  2.77519  2.98518  4.58477  3.61503
          0.02321  4.17053  4.89288  0.61958  0.77255  0.48576  0.95510
    137   3.25266  4.59255  4.86528  4.29758  3.18898  4.50160  4.83534  2.17302  4.15958  1.11276  2.09173  4.53148  4.72252  4.28685  4.28308  3.84128  3.47963  1.79006  5.20435  4.12672    153 - -
          2.68618  4.42225  2.77519  2.73123  3.46354  2.40513  3.72494  3.29354  2.67741  2.69355  4.24690  2.90347  2.73739  3.18146  2.89801  2.37887  2.77519  2.98518  4.58477  3.61503
          0.02321  4.17053  4.89288  0.61958  0.77255  0.48576  0.95510
    138   1.79466  4.79782  2.82404  1.94090  4.31166  3.30608  3.81075  3.72422  2.65702  3.33717  4.15830  2.99339  3.84716  2.97720  3.11199  2.03142  2.93949  3.33667  5.56525  4.22435    154 - -
          2.68618  4.42225  2.77519  2.73123  3.46354  2.40513  3.72494  3.29354  2.67741  2.69355  4.24690  2.90347  2.73739  3.18146  2.89801  2.37887  2.77519  2.98518  4.58477  3.61503
          0.02321  4.17053  4.89288  0.61958  0.77255  0.48576  0.95510
    139   2.58506  4.29342  3.55405  3.00981  3.47023  2.68992  3.90060  2.80129  2.93915  2.04902  2.51305  3.37662  3.96778  3.25166  3.26632  2.35763  2.84501  2.57968  4.93542  3.70537    155 - -
          2.68618  4.42225  2.77519  2.73123  3.46354  2.40513  3.72494  3.29354  2.67741  2.69355  4.24690  2.90347  2.73739  3.18146  2.89801  2.37887  2.77519  2.98518  4.58477  3.61503
          0.02321  4.17053  4.89288  0.61958  0.77255  0.48576  0.95510
    140   3.08583  4.40689  4.84510  4.29544  2.25790  4.37874  4.73777  1.57472  4.17210  1.94826  3.22641  4.44953  4.66205  4.33233  4.29141  3.72292  3.32617  1.35939  5.16135  3.94030    156 - -
          2.68618  4.42225  2.77519  2.73123  3.46354  2.40513  3.72494  3.29354  2.67741  2.69355  4.24690  2.90347  2.73739  3.18146  2.89801  2.37887  2.77519  2.98518  4.58477  3.61503
          0.02321  4.17053  4.89288  0.61958  0.77255  0.48576  0.95510
    141   2.02979  2.61338  3.45168  2.91840  3.84883  3.32836  3.92100  3.21873  2.81248  2.91084  3.77423  3.27409  3.86642  3.17756  2.28029  2.17107  2.82032  2.90332  5.22633  3.98297    157 - -
          2.68618  4.42225  2.77519  2.73123  3.46354  2.40513  3.72494  3.29354  2.67741  2.69355  4.24690  2.90347  2.73739  3.18146  2.89801  2.37887  2.77519  2.98518  4.58477  3.61503
          0.02321  4.17053  4.89288  0.61958  0.77255  0.48576  0.95510
    142   2.65037  4.69800  3.11777  2.58559  3.93396  3.48363  3.71066  2.49275  2.05906  2.95516  3.80693  2.50024  3.88815  2.89605  2.88872  2.72907  2.42763  2.99850  5.24239  3.94877    158 - -
          2.68618  4.42225  2.77519  2.73123  3.46354  2.40513  3.72494  3.29354  2.67741  2.69355  4.24690  2.90347  2.73739  3.18146  2.89801  2.37887  2.77519  2.98518  4.58477  3.61503
          0.02321  4.17053  4.89288  0.61958  0.77255  0.48576  0.95510
    143   2.21701  4.52657  2.45380  2.75014  3.73535  3.53805  3.84294  2.86807  2.75770  2.17002  3.64455  3.21159  3.95457  3.08148  3.16791  2.80094  2.89608  2.24528  5.16549  3.89225    159 - -
          2.68618  4.42225  2.77519  2.73123  3.46354  2.40513  3.72494  3.29354  2.67741  2.69355  4.24690  2.90347  2.73739  3.18146  2.89801  2.37887  2.77519  2.98518  4.58477  3.61503
          0.02321  4.17053  4.89288  0.61958  0.77255  0.48576  0.95510
    144   3.41438  4.70993  5.04893  4.51836  3.21871  4.71073  5.05993  1.60681  4.34794  0.90468  3.01922  4.76098  4.89446  4.48359  4.45935  4.09339  3.64659  2.14279  5.32906  4.22624    160 - -
          2.68618  4.42225  2.77519  2.73123  3.46354  2.40513  3.72494  3.29354  2.67741  2.69355  4.24690  2.90347  2.73739  3.18146  2.89801  2.37887  2.77519  2.98518  4.58477  3.61503
          0.02321  4.17053  4.89288  0.61958  0.77255  0.48576  0.95510
    145   1.66341  4.55765  3.29747  2.78822  4.09563  3.34088  3.86205  3.44543  2.60980  3.11452  3.96692  3.19039  3.87148  3.05878  2.16675  2.67137  2.29043  3.09965  5.38848  4.12438    161 - -
          2.68618  4.42225  2.77519  2.73123  3.46354  2.40513  3.72494  3.29354  2.67741  2.69355  4.24690  2.90347  2.73739  3.18146  2.89801  2.37887  2.77519  2.98518  4.58477  3.61503
          0.02321  4.17053  4.89288  0.61958  0.77255  0.48576  0.95510
    146   1.96741  4.60402  3.09101  2.67267  4.09875  3.30287  2.64764  3.55483  2.68845  3.18630  4.02165  3.11035  3.84730  3.04694  3.09548  1.72418  2.88008  3.18467  5.41006  4.09565    162 - -
          2.68618  4.42225  2.77519  2.73123  3.46354  2.40513  3.72494  3.29354  2.67741  2.69355  4.24690  2.90347  2.73739  3.18146  2.89801  2.37887  2.77519  2.98518  4.58477  3.61503
          0.02321  4.17053  4.89288  0.61958  0.77255  0.48576  0.95510
    147   2.02427  4.88556  3.14956  2.70898  4.40362  3.45865  3.80224  3.73109  1.30907  3.34916  4.21424  3.15279  3.95761  2.96243  2.65527  2.83485  3.06573  3.38348  5.54009  4.27566    163 - -
          2.68618  4.42225  2.77519  2.73123  3.46354  2.40513  3.72494  3.29354  2.67741  2.69355  4.24690  2.90347  2.73739  3.18146  2.89801  2.37887  2.77519  2.98518  4.58477  3.61503
          0.02321  4.17053  4.89288  0.61958  0.77255  0.48576  0.95510
    148   3.70647  5.07328  4.29292  4.08736  2.37118  4.18885  3.80078  3.67800  3.93734  3.05218  4.32316  4.11065  4.65987  4.16910  4.05541  3.80923  3.99693  3.56378  4.02310  0.58497    164 - -
          2.68618  4.42225  2.77519  2.73123  3.46354  2.40513  3.72494  3.29354  2.67741  2.69355  4.24690  2.90347  2.73739  3.18146  2.89801  2.37887  2.77519  2.98518  4.58477  3.61503
          0.19360  4.17053  1.82902  0.61958  0.77255  0.48576  0.95510
    149   2.92198  5.11574  3.28049  2.65489  4.47826  3.59727  2.51142  3.88373  1.57593  3.35205  4.19259  3.10178  3.96579  2.72398  1.84611  2.91372  3.10363  3.55066  5.42147  4.18835    165 - -
          2.68634  4.42241  2.77536  2.73098  3.46370  2.40469  3.72511  3.29370  2.67757  2.69331  4.24706  2.90363  2.73756  3.18097  2.89817  2.37903  2.77536  2.98535  4.58493  3.61418
          0.22163  1.61553        *  1.50361  0.25145  0.00000        *
//
</pre>
</td></tr></table><p>





<H2>
    Output file format
</H2>

<!-- 
        This includes example output file formats.
        This should be a detailed description and example - assume
        someone will want to parse this file and will want to know what
        happens in unusual cases - null output, errors etc. 

        If you wish to include the standard description of the avalable
	report formats, use:

        #include file="inc/reportformats.ihtml"
   -->

<b>eomegash</b> writes alignments using the default Clustal-Omega output.


<p>


<a name="output.1"></a>
<h3>Output files for usage example </h3>
<p><h3>File: twoglob.aln</h3>
<table width="90%"><tr><td bgcolor="#CCFFCC">
<pre>
&gt;HBA_HUMAN P69905 Hemoglobin subunit alpha (Alpha-globin) (Hemoglobin alpha chain)
MVLSPADKTNVKAAWGKVGAHAGEYGAEALERMFLSFPTTKTYFPHFDLSHGSAQVKGHG
KKVADALTNAVAHVDDMPNALSALSDLHAHKLRVDPVNFKLLSHCLLVTLAAHLPAEFTP
AVHASLDKFLASVSTVLTSKYR
&gt;HBA_BOVFRAG Hemoglobin subunit alpha (Alpha-globin) fragment (Hemoglobin alpha chain)
------DKGNVKAAWGKVGGHAAEYGAEALERMFLSFPTTKTYFPHFDLSHGSAQVKGHG
AKVAAALTKAVEHLDDLPGALSELSDLHAHKLRVDPVNFKLLSHSLLVTLASHLPSDFTP
AVHASLDKFLANVST-------
</pre>
</td></tr></table><p>





<H2>
    Data files
</H2>

<!-- 
        Any data files used (e.g. translation table file)
        This includes example data file formats if they are
        not obvious.

	If you wish to include the standard description of what data
	files are and how to use embossdata to inspect and retrieve
	them, use:

        #include file="inc/localfiles.ihtml"
   -->

None.














<H2>
    Notes
</H2>

<!-- 
        Restrictions.
        Interesting behaviour.
        Useful things you can do with this program.
   -->

None.







<H2>
    References
</H2>

<!-- 
        Bibliography for methods used.
   -->

[1] Johannes Soding (2005) Protein homology detection by HMM-HMM
    comparison. Bioinformatics 21 (7): 951–960.
<p>
[2] Blackshields G, Sievers F, Shi W, Wilm A, Higgins DG.  Sequence
    embedding for fast construction of guide trees for multiple
    sequence alignment.  Algorithms Mol Biol. 2010 May 14;5:21.
<p>
[3] http://www.genetics.wustl.edu/eddy/software/#squid
<p>
[4] Wilbur and Lipman, 1983; PMID 6572363
<p>
[5] Thompson JD, Higgins DG, Gibson TJ.  (1994). CLUSTAL W: improving
    the sensitivity of progressive multiple sequence alignment through
    sequence weighting, position-specific gap penalties and weight
    matrix choice. Nucleic Acids Res., 22, 4673-4680.
<p>
[6] Larkin MA, Blackshields G, Brown NP, Chenna R, McGettigan PA,
    McWilliam H, Valentin F, Wallace IM, Wilm A, Lopez R, Thompson JD,
    Gibson TJ, Higgins DG.  (2007). Clustal W and Clustal X version
    2.0. Bioinformatics, 23, 2947-2948.
<p>
[7] Kimura M (1980). "A simple method for estimating evolutionary
    rates of base substitutions through comparative studies of
    nucleotide sequences". Journal of Molecular Evolution 16: 111–120.
<p>
[8] Edgar, R.C. (2004) MUSCLE: multiple sequence alignment with high
    accuracy and high throughput.Nucleic Acids Res. 32(5):1792-1797.






<H2>
    Warnings
</H2>

<!-- 
        Potentially stupid things the program will let you do.
   -->

None.







<H2>
    Diagnostic Error Messages
</H2>

<!-- 
        Error messages specific to this program, eg:
        "FATAL xxx" - means you have not set up the xxx data using program <b>prog</b>.<p>
   -->

None.







<H2>
    Exit status
</H2>

<!-- 
        Description of the exit status for various error conditions
   -->

It always exits with status 0.








<H2>
    Known bugs
</H2>


<!-- 
        Bugs noted but not yet fixed.
   -->

None.








<!--
<H2>
    See also
</H2>
-->
<h2><a name="See also">See also</a></h2>
<table border cellpadding=4 bgcolor="#FFFFF0">
<tr><th>Program name</th>
<th>Description</th></tr>
<tr>
<td><a href="/emboss/apps/edialign.html">edialign</a></td>
<td>Local multiple alignment of sequences</td>
</tr>

<tr>
<td><a href="/emboss/apps/emma.html">emma</a></td>
<td>Multiple sequence alignment (ClustalW wrapper)</td>
</tr>

<tr>
<td><a href="eomega.html">eomega</a></td>
<td>Multiple sequence alignment (ClustalO wrapper)</td>
</tr>

<tr>
<td><a href="eomegapp.html">eomegapp</a></td>
<td>Profile with profile (ClustalO wrapper)</td>
</tr>

<tr>
<td><a href="eomegasp.html">eomegasp</a></td>
<td>Sequence with profile (ClustalO wrapper)</td>
</tr>

<tr>
<td><a href="/emboss/apps/infoalign.html">infoalign</a></td>
<td>Display basic information about a multiple sequence alignment</td>
</tr>

<tr>
<td><a href="/embassy/mse/mse.html">mse</a></td>
<td>Multiple sequence editor</td>
</tr>

<tr>
<td><a href="/emboss/apps/plotcon.html">plotcon</a></td>
<td>Plot conservation of a sequence alignment</td>
</tr>

<tr>
<td><a href="/emboss/apps/prettyplot.html">prettyplot</a></td>
<td>Draw a sequence alignment with pretty formatting</td>
</tr>

<tr>
<td><a href="/emboss/apps/showalign.html">showalign</a></td>
<td>Display a multiple sequence alignment in pretty format</td>
</tr>

<tr>
<td><a href="/emboss/apps/tranalign.html">tranalign</a></td>
<td>Generate an alignment of nucleic coding regions from aligned proteins</td>
</tr>

</table>
<!-- 
        Add any comments about other associated programs (to prepare
        data files?) that seealso doesn't find. 
   -->










<H2>
    Author(s)
</H2>

Alan Bleasby 
<br>
European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge CB10 1SD, UK

<p>
Please report all bugs to the EMBOSS bug team (emboss-bug&nbsp;&copy;&nbsp;emboss.open-bio.org) not to the original author.

<H2>
    History
</H2>





<H2>
    Target users
</H2>

<!--
        For general users, requested by one user, for EMBOSS site
        maintainers, for EMBOSS developers etc.
        eg:
        "This program is intended to be used by everyone and everything,
        from naive users to embedded scripts." 
	Which is easy to include using:
   -->

This program is intended to be used by everyone and everything, from naive users to embedded scripts.


<H2>
    Comments
</H2>

None

</BODY>
</HTML>

